src/main/java/picocli/CommandLine.java

```diff
@@ -5138,9 +5133,8 @@ public Help create(CommandSpec commandSpec, Help.ColorScheme colorScheme) {
                 return cls.cast(new LinkedHashMap<Object, Object>());
             }
             try {
-                @SuppressWarnings("deprecation") // Class.newInstance is deprecated in Java 9
-                T result = cls.newInstance();
-                return result;
```

Comment:
This `@SuppressWarnings("deprecation")` annotation suppresses the warning one gets when building the project with Java 9. I think it is better to keep it.